Source/Public/Get-AZDOPSNode.ps1

```diff
@@ -0,0 +1,24 @@
+function Get-AZDOPSNode {
+    [CmdletBinding()]
+    param (
+        [Parameter()]
+        [string]$Organization,
```

Comment:
If no organization is given the URL will not work. Please change this to mandatory, or use the $org['organization'] variable in the URL as that one will always be populated.

Reply:
Would probalby help if i read the code right though.. sorry.. :/